src/OpenApi/CodeGenerator/CodeGenerator.cs

```diff
@@ -0,0 +1,190 @@
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers;
+
+namespace CodeGenerator;
+
+public class App
+{
+    public static void Main(string[] args)
+    {
+        if (args.Length == 0)
+        {
+            Console.WriteLine("No file path was found!");
+            return;
+        }
+        var document = ReadJson(args[0]);
+        var paths = document.Paths;
+
+        if (paths is null || paths.Count == 0)
+        {
+            Console.WriteLine("No path was found in the schema!");
+            return;
+        }
+
+        RuntimeTextTemplate2 page;
```

Comment:
Nit: Can we rename this to something clearer?

```suggestion
        MinimalApiTemplate page;
```